src/invidious/comments.cr

```diff
@@ -552,7 +552,10 @@ def content_to_comment_html(content)
       if url = run["navigationEndpoint"]["urlEndpoint"]?.try &.["url"].as_s
         url = URI.parse(url)
 
-        if !url.host || {"m.youtube.com", "www.youtube.com", "youtu.be"}.includes? url.host
+        if !url.host
+        elsif url.host == "youtu.be"
+          url = "/watch?v=#{url.request_target.lstrip('/')}"
+        elsif {"m.youtube.com", "www.youtube.com"}.includes? url
           if url.path == "/redirect"
```

Comment:
Afaik `youtu.be` doesn't actually have a `/redirect` path. It should be fine to remove handling for `youtu.be` here.